e2e/wdio/core/tests/standard-list.e2e-spec.ts

```diff
@@ -0,0 +1,422 @@
+            checkElementStates(navigationList + listItems, 'hover', 'navigation list item');
+            checkElementStates(navigationList + listItems, 'focus', 'navigation list item');
+            checkElementStates(navigationList + listItems, 'active', 'navigation list item');
+        });
+    });
+
+    describe('navigation indicator list examples', function() {
+        it('check list item states', () => {
+            checkElementStates(navigationIndicatorList + listItems, 'hover', 'navigation indicator list item');
+            checkElementStates(navigationIndicatorList + listItems, 'focus', 'navigation indicator list item');
+            checkElementStates(navigationIndicatorList + listItems, 'active', 'navigation indicator list item');
+        });
+    });
+
+    describe('action list examples', function() {
+        it('check list item states', () => {
+            checkElementStates(actionList + listItems, 'hover', 'action list item');
+            checkElementStates(actionList + listItems, 'focus', 'action list item');
+            checkElementStates(actionList + listItems, 'active', 'action list item');
+        });
+
+        it('should check show more button functionality', () => {
+            const startItemsCount = getElementArrayLength(actionList + listItems);
+
+            click(actionList + button);
+            waitForNotDisplayed(actionList + busyIndicator);
+
+            expect(getElementArrayLength(actionList + listItems)).not.toEqual(startItemsCount);
+        });
+    });
+
+    describe('filter and sort list examples', function() {
+        afterEach(() => {
+            refreshPage();
+            waitForElDisplayed(standardListPage.title);
+        }, 1);
+
+        it('check list item states', () => {
+            checkElementStates(filterAndSortList + listItems, 'hover', 'filter and sort list item');
+            checkElementStates(filterAndSortList + listItems, 'focus', 'filter and sort list item');
+            checkElementStates(filterAndSortList + listItems, 'active', 'filter and sort list item');
+        });
+
+        it('should check ability to search', () => {
+            scrollIntoView(filterAndSortList);
+            click(filterAndSortList + searchBar);
+            sendKeys('apple');
+            const resultCount = getElementArrayLength(filterAndSortList + listItems);
+            for (let i = 0; i < resultCount; i++) {
+                expect(getText(filterAndSortList + listItems, i).toLowerCase()).toContain('apple');
+            }
+        });
+
+        it('should check the ability to clear search', () => {
+            const startItemCount = getElementArrayLength(filterAndSortList + listItems);
+
+            scrollIntoView(filterAndSortList);
+            click(filterAndSortList + searchBar);
+            sendKeys('apple');
+
+            expect(getElementArrayLength(filterAndSortList + listItems)).not.toEqual(startItemCount);
+
+            click(filterAndSortList + button);
+
```

Comment:
Same as above

Reply:
Also, saw the same for other scenarios - @Seamoo13 Is it ok?

Reply:
https://www.notion.so/valorlabs/Automated-Tests-Styleguide-ad092cdea96d4d36a6fa24328688da43#347d5031a2d140e7b45ddd05e3b39205

following AAA style per styleguide